actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -328,6 +330,7 @@ def favicon_link_tag(source = "favicon.ico", options = {})
       # * <tt>:crossorigin</tt>  - Specify the crossorigin attribute, required to load cross-origin resources.
       # * <tt>:nopush</tt>  - Specify if the use of server push is not desired for the resource. Defaults to +false+.
       # * <tt>:integrity</tt> - Specify the integrity attribute.
+      # * <tt>:nonce</tt>  - When set to true, adds an automatic nonce value if you have Content Security Policy enabled.
```

Comment:
Why is it opt-in if it's required by some browsers?

Reply:
The rationale is generally in the PR: https://github.com/rails/rails/pull/32607#issuecomment-531550756

I'm no expert on CSP, but I think here the rationale doesn't apply?

Reply:
Assuming that browsers don't do anything with resources (specifically scripts) included with a `link`, other than downloading them, I tend to agree with you. I'll see if I can make that work.